espanol/transcender/transcender.md

```diff
@@ -5,4 +5,8 @@ su existencia el Consejero. Una de las últimas figuras que, por así decir, des
 ya no era mas que una masa de colores amorfa que aún se diferenciaba del "fondo".
 
 Pero aún persistian sus pensamientos. Debería desprenderse de esos también, si quería transcender 
-debidamente.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.